empyrical/tests/test_stats.py

```diff
@@ -1048,6 +1048,90 @@ def empyrical(self):
         )
 
 
+class Test2DStats(TestCase):
+    """
+    Tests for functions that are capable of outputting a DataFrame.
+    """
+
+    input_one = [np.nan, 0.01322056, 0.03063862, -0.01422057,
+                 -0.00489779, 0.01268925, -0.03357711, 0.01797036]
```

Comment:
Should we add the edge cases that we have for 1D, like empty and nans in the final position?